### PR TITLE
feat(thirdparty): add verifications resource calls

### PR DESCRIPTION
**Pull request by lewisdaly**
part of mojaloop/project#2372

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -369,6 +369,49 @@ declare namespace SDKStandardComponents {
...
+        /**
+         * @function postThirdpartyRequestsVerifications
+         * @description
+         *   Executes a `POST /thirdpartyRequests/verifications` request
+         * @param {Object} thirdpartyRequestsTransactionsBody The thirdpartyRequestsVerificationsBody
+         * @param {string} destParticipantId The id of the destination participant, in this case, a DFSP
+         * @returns {Promise<object>} JSON response body if one was received
+         */
+        postThirdpartyRequestsVerifications(
+            thirdpartyRequestsVerificationsBody: tpAPI.Schemas.ThirdpartyRequestsVerificationsPostRequest,
```
Comment (kleyow) [inline src/index.d.ts:382]: I think you should address the schema changes needed for the fido assertion before merging this, you'll have to push another build after the changes other wise.

https://github.com/mojaloop/api-snippets/pull/101/files#r684868466

**Review by kleyow — CHANGES_REQUESTED** [review]
also tests are needed.

File: src/index.d.ts (hunk 1 above)
Comment (lewisdaly) [reply src/index.d.ts:382]: Yeah I think you're right... I've been putting them off for to long 🤣

**Comment by lewisdaly** [comment]
Do you mind taking a look at this one then? https://github.com/mojaloop/pisp-project/pull/149

File: src/index.d.ts (hunk 1 above)
Comment (lewisdaly) [reply src/index.d.ts:382]: Done!

**Review by kleyow — APPROVED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**